index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> serving as possible extension points. While
+some implementers signaled interest in these, their inclusion in this specification
+was considered to be premature; these extension points might be more
+formally defined in future versions of this specification. While these extension
+points are reserved in the base context, their usage is undefined by this
+specification and implementers are cautioned that usage of these properties is
+considered experimental.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `render`, `refreshService` and `confirmationMethod`.
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Reserved&nbsp;Property</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>
+`example`
+              </td>
+              <td>
+An example entry for a reserved property.
+              </td>
+            </tr>
+
+          </tbody>
+        </table>
+
+        <p>
+A list of specifications that utilize the extension points defined in this
+specification, as well as the reserved extension points listed above, can
+be found in the Verifiable Credentials Specifications Directory
+[[VC-SPECS-DIR]].
```

Comment:
I think it important to clarify the nature of the relationship between this spec and the vc-specs-directory and to provide normative guidance.

```suggestion
A non-normative list of specifications that utilize the extension points defined
in this specification, as well as the reserved extension points, can be found in
the Verifiable Credentials Specifications Directory [[VC-SPECS-DIR]]. Items in
the directory referring to reserved extension points SHOULD be treated as
experimental.
```

Reply:
I like this language.

Reply:
I'm fine with Ted's version